apps/ex_nvr_web/assets/js/webrtc.js

```diff
@@ -0,0 +1,54 @@
+import { Socket } from "phoenix"
+import { WebRTCEndpoint } from "@jellyfish-dev/membrane-webrtc-js"
+
+function getWebSocketEndpoint() {
+    const protocol = window.location.protocol === 'https:' ? 'wss://' : 'ws://'
+    const hostname = window.location.hostname
+    const port = window.location.port ? `:${window.location.port}` : ''
+    
+    return `${protocol}${hostname}${port}`
+}
+
+window.onload = function (_event) {
+    const player = document.getElementById("webRtcPlayer")
+    const deviceId = player.dataset.device
+    const webrtc = new WebRTCEndpoint()
+    
+    const socket = new Socket(getWebSocketEndpoint() + "/socket", {params: {}})
+    const channel = socket.channel(`device:${deviceId}`)
+
+    channel.on("media_event", ({ data }) => {
+        console.log("Received event from channel: ", data)
```

Comment:
This logs are used for debugging, I don't think we'll need them on production.
I think it's better to show this logs somewhere on the UI (hidden by default), probably overlayed on the video player.